conda/core/prefix_data.py

```diff
@@ -112,7 +113,7 @@ def _get_json_fn(self, prefix_record):
             )
         return fn + ".json"
 
-    def insert(self, prefix_record):
+    def insert(self, prefix_record, dump_auth=False):
```

Comment:
```suggestion
    def insert(self, prefix_record, include_tokens=False):
```

I think this is a little more descriptive.

Reply:
To stay consistent with the argument added in (my review of) #13936:

```suggestion
    def insert(self, prefix_record, remove_auth=True):
```